template/package.json

```diff
@@ -4,19 +4,30 @@
   "description": "{{ description }}",
   "author": "{{ author }}",
   "license": "{{ license }}",
-  {{#if_eq license "UNLICENSED"}}
   "private": true,
-  {{/if_eq}}
   "scripts": {
     "build": "roc build",
     "dev": "roc dev",
     "start": "roc start"
   },
   "dependencies": {
-    "roc-package-web-app-react": "^1.0.0-beta.3"
+    "babel-eslint": "^7.1.0",
+    "eslint": "^3.6.0",
```

Comment:
It looks like eslint was added as a dependency, but there is no way to run linting other than using `node_modules/.bin/eslint` (directly or in npm scripts) or other tools, that can work with local eslint package (IDEs and text editors linter plugins).

Maybe it's better to move common eslint setup and React-specific configs into its own plugins, providing roc commands for linting source files?